R/update.hydromad.R

```diff
@@ -3,6 +3,33 @@
 ## Copyright (c) Felix Andrews <felix@nfrac.org>
 ##
 
+#' Placeholder title for absorbScale
+#'
+#' Description here.
+#'
+#' @importFrom zoo as.zooreg
+#' @importFrom stats predict
+#'
+#' @aliases absorbScale
+#'
+#' @param object Placeholder
+#' @param gain Placeholder
+#' @param ... Placeholder
+
+
+#' @export
+absorbScale <- function(object, gain, ...) {
+  UseMethod("absorbScale")
+}
+
+
+#' @export
+absorbScale.hydromad <- function(object, gain, ...) {
+  return(NULL)
+}
+
+
+#' @export
 update.hydromad <-
```

Comment:
When are we meant to use an rdname and when are we not?

Reply:
`@rdname` is used to document a group of functions in the one .Rd document when those functions have some similar but also some different parameters.

[https://cran.r-project.org/web/packages/roxygen2/vignettes/rd.html](https://cran.r-project.org/web/packages/roxygen2/vignettes/rd.html)

An example of its usage is here:

[https://gist.github.com/jefferys/b79fe87314b0dc72fec9](https://gist.github.com/jefferys/b79fe87314b0dc72fec9)